README.md

```diff
@@ -30,6 +30,16 @@ Install:
 - [Zig](https://ziglang.org) (version 0.11.0)
 - [DOSBox](https://www.dosbox.com)
 
+Add DOSBox to the path:
+
+- Mac: Add `export PATH="$PATH:/applications/dosbox.app/contents/macos/"` to your `~/.zshrc`.
+- Windows: https://stackoverflow.com/questions/9546324/adding-a-directory-to-the-path-environment-variable-in-windows
+
+Add CWSDPMI:
+
+- [Download the binary](https://sandmann.dotster.com/cwsdpmi/csdpmi7b.zip)
```

Comment:
`deps/cwsdpmi` is a git submodule, so it should be `git submodule init && git submodule update` after the first clone to get it.

Reply:
I had completely missed that. Fixed.